src/appendix/glossary.md

```diff
@@ -61,7 +61,7 @@ tcx                     |  the "typing context", main data structure of the comp
 trait reference         |  the name of a trait along with a suitable set of input type/lifetimes ([see more](./traits/goals-and-clauses.html#trait-ref))
 token                   |  the smallest unit of parsing. Tokens are produced after lexing ([see more](the-parser.html)).
 [TLS]                   |  Thread-Local Storage. Variables may be defined so that each thread has its own copy (rather than all threads sharing the variable). This has some interactions with LLVM. Not all platforms support TLS.
-trans                   |  the code to translate MIR into LLVM IR.
+codegen                 |  the code to translate MIR into LLVM IR.
```

Comment:
It might be a good idea to still have an entry for trans in the glossary in case people stumble upon old issues or PRs.